cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -47,87 +50,134 @@ <h1>{{ page.title }}</h1>
+                        {% endfor %}
+                </dd>
+            {% endif %}
         </dl>
     </section>
 
     <section class="block
                     block__padded-top
-                    block__border-top">
-        <h2>Job Description</h2>
-        {{ parse_links( page.description ) | safe }}
-    </section>
+                    block__border-top
+                    block__flush-bottom
+                    vacancy-announcement_description">
+        <div data-qa-hook="expandable"
+             data-read-more="true"
+             class="o-expandable
+                    o-expandable__read-more">
+            <div class="o-expandable_content" id="job-description-expandable">
+                <div class="o-expandable_content-animated">  
+                    <h4>Summary</h4>
+                    {{ parse_links( page.description ) | safe }}
+                    {% if page.responsibilities %}
+                        <div class="block block__sub">
+                            <h4>Responsibilities</h4>
+                            {{ parse_links( page.responsibilities ) | safe }}
+                        </div>
+                    {% endif %}
+        
+                    {% if page.travel_required %}
+                        <div class="block block__sub">
+                            <h4>Travel required</h4>
+                            {% if page.travel_details %}
+                            {{ parse_links( page.travel_details ) | safe }}
+                            {% else %}
+                            Yes
+                            {% endif %}
+                        </div>
+                    {% endif %}
 
-    <section class="block">
-        <em>
-            The Consumer Financial Protection Bureau (CFPB) is an equal
-            opportunity employer and seeks to create and maintain a vibrant
-            and diverse workforce. Women, minorities, veterans, and
-            people with disabilities are encouraged to apply.
-        </em>
-    </section>
+                    {% if page.additional_section_title and page.additional_section_content %}
+                        <div class="block block__sub">
+                            <h4>{{ page.additional_section_title }}</h4>
+                            {{ parse_links( page.additional_section_content ) | safe }}
+                        </div>
+                    {% endif %}
 
+                    {% if page.service_type and page.job_length %}
+                        <div class="block block__sub">
+                            <h4>Position type</h4>
+                            {{- page.service_type }}, {{ page.job_length | lower -}}
+                        </div>
+                    {% endif %}
+                </div>
+            </div>
+            <a class="o-expandable_target o-expandable_link a-link" title="Expand content">
+                <span class="a-link_text">
```

Comment:
`a-link_text` limits the underline to just the text and not the icon for instances when a link has an icon. Not sure what the best solution is here since our [recommended expandables pattern](https://cfpb.github.io/capital-framework/components/cf-expandables/#default-state) is to include the icon inside the text span but you may want to take a second look at this.

Reply:
Looks like that wasn't necessary -- updated in https://github.com/cfpb/cfgov-refresh/pull/3784/commits/91043aa558504728cd662a5faca32bea76e71074.